shared/modules/persistence_module.go

```diff
@@ -9,6 +9,7 @@ type PersistenceModule interface {
 	Module
 	NewRWContext(height int64) (PersistenceRWContext, error)
 	NewReadContext(height int64) (PersistenceReadContext, error)
+	ResetContext() error
```

Comment:
TODO remove from interface once moved to test function and/or re-scope it at the context level

Reply:
Added.